ekncontent/ekncontent/eknc-article-object-model.c

```diff
@@ -321,6 +347,23 @@ eknc_article_object_model_get_authors (EkncArticleObjectModel *self)
 }
 
 /**
+ * eknc_article_object_model_get_temporal_coverage:
```

Comment:
Noticed one more thing when running the tests! You should add this to `ekncontent/docs/reference/ekncontent/eos-knowledge-content-sections.txt`.